src/status_im/chat/handlers.cljs

```diff
@@ -100,7 +100,7 @@
 (register-handler :cancel-command
   (fn [{:keys [current-chat-id] :as db} _]
     (-> db
-        (dissoc :canceled-command)
+        (assoc :canceled-command nil)
```

Comment:
The database should be readable and transparent, there shouldn't be "hidden" fields, that's why i added all fields on init